dictionary/get_un_names.R

```diff
@@ -7,30 +7,210 @@ source(here::here('dictionary/utilities.R'))
+                                 un.name.fr),
+             un.name.es = ifelse(un.name.es == 'Gambia (República de)',
+                                 'Gambia',
+                                 un.name.es),
+             # Guinea-Bissau
+             un.name.en = ifelse(un.name.en == 'Guinea Bissau',
+                                 'Guinea-Bissau',
+                                 un.name.en),
+             un.name.es = ifelse(un.name.es == 'Guinea Bissau',
+                                 'Guinea-Bissau',
+                                 un.name.es),
+             # Iran
+             un.name.fr = ifelse(un.name.fr == 'Iran (République islamique d\')',
+                                 'Iran (République islamique d’)',
+                                 un.name.fr),
+             un.name.zh = ifelse(un.name.zh == '伊朗伊斯兰共和国',
+                                 '伊朗（伊斯兰共和国）',
+                                 un.name.zh),
+             # Micronesia
+             un.name.zh = ifelse(un.name.zh == '密克罗尼西亚联邦',
+                                 '密克罗尼西亚 (联邦)',
+                                 un.name.zh),
+             # Niger
+             un.name.es = ifelse(un.name.es == 'Niger',
+                                 'Níger',
+                                 un.name.es),
+             # North Macedonia
+             un.name.ar = ifelse(un.name.ar == 'جمهورية مقدونيا الشمالية',
+                                 'مقدونيا الشمالية',
+                                 un.name.ar),
+             # South Sudan
+             un.name.ar = ifelse(un.name.ar == 'جمهورية جنوب السودان',
+                                 'جنوب السودان',
+                                 un.name.ar),
+             # Timor-Leste
+             un.name.ar = ifelse(un.name.ar == 'تيمور- ليشتي',
+                                 'تيمور-ليشتي',
+                                 un.name.ar),
+             # Trinidad and Tobago
+             un.name.es = ifelse(un.name.es == 'Trinidad y Tabago',
+                                 'Trinidad y Tobago',
+                                 un.name.es),
+             # United Kingdom
+             un.name.fr = ifelse(un.name.fr == 'Royaume-Uni de Grande-Bretagne et d\'Irlande du Nord',
+                                 'Royaume-Uni de Grande-Bretagne et d’Irlande du Nord',
+                                 un.name.fr),
+             # United States
+             un.name.fr = ifelse(un.name.fr == 'États-Unis d\'Amérique',
+                                 'États-Unis d’Amérique',
+                                 un.name.fr),
+             # Venezuela
+             un.name.en = ifelse(un.name.en == 'Venezuela, Bolivarian Republic of',
+                                 'Venezuela (Bolivarian Republic of)',
+                                 un.name.en),
+             un.name.zh = ifelse(un.name.zh == '委内瑞拉玻利瓦尔共和国',
+                                 '委内瑞拉（玻利瓦尔共和国）',
+                                 un.name.zh),
```

Comment:
Turns out there are a lot of mistakes on the UN website! Most are minor, but still.